src/payment/strategies/bolt/bolt-app-payment-strategy.ts

```diff
@@ -0,0 +1,106 @@
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { InvalidArgumentError, MissingDataError, MissingDataErrorType, NotInitializedError, NotInitializedErrorType } from '../../../common/error/errors';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentMethodCancelledError } from '../../errors';
+import { NonceInstrument } from '../../payment';
+import PaymentActionCreator from '../../payment-action-creator';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import { BoltCheckout, BoltTransacion } from './bolt';
+import BoltScriptLoader from './bolt-script-loader';
+
+export default class BoltAppPaymentStrategy implements PaymentStrategy {
+    private _boltClient?: BoltCheckout;
+    private _methodId!: string;
```

Comment:
what's the `!` for?

Reply:
It allows the variable not to be initialized and not to be assigned in the constructor.